Makefile

```diff
@@ -190,11 +184,16 @@ chart: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(shell echo ${IMG} | sed 's/\(:.*\)v/\1/')
 	$(KUSTOMIZE) build config/default > chart/templates/operator.yaml
 
-.PHONY: chart-push
-chart-push: chart ## push helm chart
-	helm package ./chart --destination . --version ${IMAGE_TAG} --app-version ${IMAGE_TAG}
-	helm push ./*.tgz oci://${KO_DOCKER_REPO}
-	rm ./*.tgz
+# this is used for local testing - since the image is locally built it needs to be loaded also on the WEC cluster(s)
+.PHONY: kind-load-image
+kind-load-image: 
+	kind load --name ${KIND_HOSTING_CLUSTER} docker-image ${IMG}
+	kind load --name ${DEFAULT_WEC1_CONTEXT} docker-image ${IMG}
+	kind load --name ${DEFAULT_WEC2_CONTEXT} docker-image ${IMG}
```

Comment:
This seems specific to one usage. Wouldn't it be better to have a `make` variable saying where to load the image into?

Reply:
Not sure why loading the image is part of a Makefile at all ...

Reply:
Yes, I think this would be better as a shell script.

Reply:
This was mirroring to some extend the `kind-load-image` we have in the KS Makefile. 
@MikeSpreitzer are you suggesting to create a script that takes as input a list of clusters to replace this?

Reply:
More likely a script that loads into just one cluster, and gets invoked as needed.

Reply:
I'm fine doing that in following PR

Reply:
me too